### PR TITLE
Propagators API minor clarifications

### PR DESCRIPTION
**Pull request by lmolkova**
Clarifies that `Setter` and `Getter` are arguments on `Extract` and `Inject` and not separate members of `HttpTextFormat`

### CODE REVIEW

File: specification/api-propagators.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ Table of Contents
 - [HTTP Text Format](#http-text-format)
   - [Fields](#fields)
   - [Inject](#inject)
-  - [Setter](#setter)
-    - [Put](#put)
+    - [Setter](#setter)
+      - [Put](#put)
```
Comment (SergeyKanzhelev) [inline specification/api-propagators.md]: maybe call it "Put callback" or "Put lambda" for clarity?

Comment (SergeyKanzhelev) [reply specification/api-propagators.md]: than maybe call it "Put argument"

**Comment by lmolkova** [comment]
could someone please merge it?

**Comment by SergeyKanzhelev** [comment]
Active for more than 2 busines days and have enough approvals. Merging

**Pull request merged**